src/components/loader/index.ts

```diff
@@ -0,0 +1 @@
+export * from './circle-loader';
```

Comment:
if Loader can accept children, it should be exported because it has sense actually

Reply:
agree